Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.2.1.plan

```diff
@@ -0,0 +1,157 @@
+{
+    "timeout": 1800,
+    "count": 1,
+    "local_git_archive": "PerformanceTests/MotionMark@86daf4fcece9d2dafe67ef0de1e23bda72b56378",
+    "github_source": "https://github.com/WebKit/WebKit/tree/86daf4fcece9d2dafe67ef0de1e23bda72b56378/PerformanceTests/MotionMark",
+    "webserver_benchmark_patch": "data/patches/webserver/MotionMark.patch",
+    "webdriver_benchmark_patch": "data/patches/webdriver/MotionMark.patch",
+    "entry_point": "index.html",
+    "config": {
+        "orientation": "landscape"
+    },
+    "output_file": "motionmark.result",
+    "github_subtree": {
+        "url": "https://api.github.com/repos/WebKit/WebKit/git/trees/23431f74c4e13875561a7778e9d98628d4293ea0",
+        "truncated": false,
+        "tree": [
```

Comment:
Do we have to list all the files in the plan? Can't Github API help us enumerating all files under a a certain directory? Otherwise we have to remember to update this tree if we add/remove a file.

Reply:
We can, but there is 60 requests/hour limit on `api.github.com`. We are trying to avoid hitting it so that by caching the response here since the response for a given committed commit should never change.
The plan file can work without `github_subtree`, but engineer will be effectively subject to "no more than 60 times per hour" limit.
`github_subtree` field can be generated using `plan_file_optimizer.py`